### PR TITLE
params: remove Infura bootnode

### PR DESCRIPTION
**Pull request by ncocchiaro**
Infura would like to retire their Rinkeby bootnode given the deprecation of the network that happened with the Merge. Before we can do that, we should make sure that Geth no longer refers to the bootnode enode among the ones hardcoded in the params package. This PR intends to address that.

### CODE REVIEW

**Review by holiman — APPROVED** [review]
LGTM

**Comment by holiman** [comment]
We need to clean up a bit more
cc @pauljickling -- the ropsten bootnodes are decommissioned as of dec 14, right?

File: params/bootnodes.go [hunk 1]
```diff
@@ -35,8 +35,6 @@ var MainnetBootnodes = []string{
 // RopstenBootnodes are the enode URLs of the P2P bootstrap nodes running on the
 // Ropsten test network.
 var RopstenBootnodes = []string{
-	"enode://30b7ab30a01c124a6cceca36863ece12c4f5fa68e3ba9b0b51407ccc002eeed3b3102d20a88f1c1d3c3154e2449317b8ef95090e77b312d5cc39354f86d5d606@52.176.7.10:30303",    // US-Azure geth
-	"enode://865a63255b3bb68023b6bffd5095118fcc13e79dcf014fe4e47e065c350c7cc72af2e53eff895f11ba1bbb6a2b33271c1116ee870f266618eadfc2e78aa7349c@52.176.100.77:30303",  // US-Azure parity
 	"enode://6332792c4a00e3e4ee0926ed89e0d27ef985424d97b6a45bf0f23e51f0dcb5e66b875777506458aea7af6f9e4ffb69f43f3778ee73c81ed9d34c51c4b16b0b0f@52.232.243.152:30303", // Parity
```
Comment (holiman) [inline params/bootnodes.go:38]: This one seems dead to me 
```
./devp2p rlpx ping "enode://6332792c4a00e3e4ee0926ed89e0d27ef985424d97b6a45bf0f23e51f0dcb5e66b875777506458aea7af6f9e4ffb69f43f3778ee73c81ed9d34c51c4b16b0b0f@52.232.243.152:30303"
```

File: params/bootnodes.go [hunk 2]
```diff
@@ -35,8 +35,6 @@ var MainnetBootnodes = []string{
...
 	"enode://94c15d1b9e2fe7ce56e458b9a3b672ef11894ddedd0c6f247e0f1d3487f52b66208fb4aeb8179fce6e3a749ea93ed147c37976d67af557508d199d9594c35f09@192.81.208.223:30303", // @gpip
```
Comment (holiman) [inline params/bootnodes.go:39]: This one also seems dead

Comment (holiman) [reply params/bootnodes.go:39]: Not quite dead, it does host an nginx http://192.81.208.223/

File: params/bootnodes.go [hunk 3]
```diff
@@ -54,7 +52,6 @@ var SepoliaBootnodes = []string{
 // Rinkeby test network.
 var RinkebyBootnodes = []string{
 	"enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303", // IE
```
Comment (holiman) [inline params/bootnodes.go:54]: This one is ok (but _very_ old)
```
$ ./devp2p rlpx ping "enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303"
{Version:5 Name:Geth/v1.10.15-unstable-356bbe34/linux-amd64/go1.17.5 Caps:[eth/66 les/2 les/3 les/4 snap/1] ListenPort:0 ID:[162 74 199 197 72 78 244 237 12 94 178 211 102 32 186 78 74 161 59 140 132 104 78 27 74 171 12 235 234 42 228 92 180 211 117 183 126 171 86 81 109 52 191 189 60 26 131 63 197 18 150 255 8 75 119 11 148 251 144 40 196 210 92 207] Rest:[]}
```

File: params/bootnodes.go [hunk 4]
```diff
@@ -54,7 +52,6 @@ var SepoliaBootnodes = []string{
...
-	"enode://343149e4feefa15d882d9fe4ac7d88f885bd05ebb735e547f12e12080a9fa07c8014ca6fd7f373123488102fe5e34111f8509cf0b7de3f5b44339c9f25e87cb8@52.3.158.184:30303",  // INFURA
 	"enode://b6b28890b006743680c52e64e0d16db57f28124885595fa03a562be1d2bf0f3a1da297d56b13da25fb992888fd556d4c1a27b1f39d531bde7de1921c90061cc6@159.89.28.211:30303", // AKASHA
```
Comment (holiman) [inline params/bootnodes.go:55]: also ok
```
$ ./devp2p rlpx ping "enode://b6b28890b006743680c52e64e0d16db57f28124885595fa03a562be1d2bf0f3a1da297d56b13da25fb992888fd556d4c1a27b1f39d531bde7de1921c90061cc6@159.89.28.211:30303"
{Version:5 Name:Geth/v1.10.25-stable-69568c55/linux-amd64/go1.18.5 Caps:[eth/66 eth/67 les/2 les/3 les/4 snap/1] ListenPort:0 ID:[182 178 136 144 176 6 116 54 128 197 46 100 224 209 109 181 127 40 18 72 133 89 95 160 58 86 43 225 210 191 15 58 29 162 151 213 107 19 218 37 251 153 40 136 253 85 109 76 26 39 177 243 157 83 27 222 125 225 146 28 144 6 28 198] Rest:[]}
```

**Review by holiman — COMMENTED** [review]
Seems like we might aswell wipe the ropsten bootnodes fully (?)

**Comment by pauljickling** [comment]
> We need to clean up a bit more cc @pauljickling -- the ropsten bootnodes are decommissioned as of dec 14, right?

Correct

**Comment by karalabe** [comment]
@ncocchiaro The testnet murder deadline is Q3 2023 tho'

**Comment by ncocchiaro** [comment]
@karalabe thanks, we're aware, but, since continuing to maintain this comes at a cost to us, we'd like to move forward now regardless. FWIW, we cleared this plan with the EF as well before proposing it here.

**Pull request merged**